devise.rb

```diff
@@ -4,23 +4,24 @@
 ########################################
 inject_into_file 'Gemfile', before: 'group :development, :test do' do
   <<~RUBY
-    gem 'devise'
-
     gem 'autoprefixer-rails', '10.2.5'
+    gem 'devise'
     gem 'font-awesome-sass'
     gem 'simple_form'
+    
   RUBY
 end
 
-inject_into_file 'Gemfile', after: 'group :development, :test do' do
+inject_into_file 'Gemfile', after: 'gem "debug", platforms: %i[ mri mingw x64_mingw ] ' do
```

Comment:
Just found an extra space that wouldn't add the gems here ⚠️

```suggestion
inject_into_file 'Gemfile', after: 'gem "debug", platforms: %i[ mri mingw x64_mingw ]' do
```

Reply:
🙏